plugins/org.jkiss.dbeaver.ext.h2/plugin.xml

```diff
@@ -87,7 +91,8 @@
                         dialect="h2">
 
                     <file type="jar" path="maven:/com.h2database:h2:RELEASE[2.1.210]" bundle="!drivers.h2_v2"/>
-                    <file type="jar" path="drivers/h2_v2" bundle="drivers.h2_v2"/>
```

Comment:
Was it removed intentionally?

Reply:
Yep. It was my previous mistake. We do not have this folder in drivers.